src/tests/ftest/pool/destroy_tests.py

```diff
@@ -1,746 +1,631 @@
+
+        uid = os.geteuid()
+        gid = os.getegid()
+
+        # Create pool with doasctl
+        create_cmd = ('{0} create-pool -m {1} -u {2} -g {3} -s {4}'
+                      .format(self.daosctl, '0731', uid, gid, goodsetid))
+        uuid_str = """{0}""".format(process.system_output(create_cmd))
+        print("uuid is {0}\n".format(uuid_str))
+        # Check if pool exists
+        self.assertTrue(
+            check_pool_files(
+                self.log, hostlist_servers, uuid_str.lower()),
+            "ERROR: Pool data not detected on servers after create")
+        self.log.info("Pool %s with %s was successfully created",
+                      uuid_str, goodsetid)
+        # delete pool
         try:
-            # use the uid/gid of the user running the test, these should
-            # be perfectly valid
-            uid = os.geteuid()
-            gid = os.getegid()
-
-            create_cmd = ('{0} create-pool -m {1} -u {2} -g {3} -s {4}'
-                          .format(daosctl, 0x731, uid, gid, goodsetid))
-            uuid_str = """{0}""".format(process.system_output(create_cmd))
-            print ("uuid is {0}\n".format(uuid_str))
-
-            exists = check_for_pool.check_for_pool(host, uuid_str)
-            if exists != 0:
-                self.fail("Pool {0} not found on host {1}.\n"
-                          .format(uuid_str, host))
-
-            delete_cmd = ('{0} destroy-pool -i {1} -s {2}'.format(daosctl,
-                                                                  uuid_str,
-                                                                  badsetid))
-
-            process.system(delete_cmd)
-
-            # the above command should fail resulting in an exception so if
-            # we get here the test has failed
-            self.fail("Pool {0} found on host {1} when not expected.\n"
-                      .format(uuid_str, host))
-
-        except Exception as _excep:
-            # expecting an exception, but now need to
-            # clean up the pool for real
             delete_cmd = ('{0} destroy-pool -i {1} -s {2}'
-                          .format(daosctl, uuid_str, goodsetid))
-            process.system(delete_cmd)
-
-        # no matter what happens shutdown the server
-        finally:
-            if self.agent_sessions:
-                agent_utils.stop_agent(self.agent_sessions)
-            server_utils.stop_server(hosts=self.hostlist_servers)
-            os.remove(hostfile_servers)
-
+                          .format(self.daosctl, uuid_str, badsetid))
+            rc = process.system(delete_cmd)
+            if rc == 0:
+                self.log.info(
+                    "destroy-pool destroyed the wrong group: %s", badsetid)
+                self.fail(
+                    "destroy-pool was expected to fail but PASSED")
+        except process.CmdError as _:
+            self.log.info(
+                "destroy-pool failed as expected with group_name = {}".format(
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments

Reply:
fixed
self.log.info(
                "destroy-pool failed as expected with group_name = %s",
                badsetid)